h2o-r/h2o-package/R/frame.R

```diff
@@ -2700,8 +2665,7 @@ h2o.ncol <- function(x) {
 #' @seealso \code{\link[base]{range}} for the base R implementation.
 #' @export
 h2o.range <- function(x,na.rm = FALSE,finite = FALSE) {
-  if( is.H2OFrame(x) ) range(x,na.rm,finite)
-  else range(x,na.rm,finite)
+  range(x,na.rm = na.rm,finite)
```

Comment:
`finite` argument is probably not used, see [range.H2OFrame](https://github.com/h2oai/h2o-3/blob/97e4d2998ae8cfb7f769bf946f9ac51c09eb0f0f/h2o-r/h2o-package/R/frame.R#L3301). Will create separate issue, as this is not directly related to this PR.